packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -118,9 +118,17 @@ def test_pod_logs(dcos_api_session):
 
     with dcos_api_session.marathon.deploy_pod_and_cleanup(pod_definition):
         url = get_task_url(dcos_api_session, pod_id)
+        url_parts = url.split('/')
+
         check_log_entry('STDOUT_LOG', url + '?filter=STREAM:STDOUT', dcos_api_session)
         check_log_entry('STDERR_LOG', url + '?filter=STREAM:STDERR', dcos_api_session)
 
+        # check app logs download
+        # the very last part in url is container_id
+        response = dcos_api_session.get(url + '/download?limit=10&postfix=stdout')
```

Comment:
FYI when you have query strings and want to manipulate them programmatically, you can now, do the following:
response = cluster.get(url+'/download', query='limit=10&postfix=stdout') Probably not the most useful thing here, but just something to know in general